PocketKit/Sources/Sync/RemoteMapping/SlateLineup+remoteMapping.swift

```diff
@@ -29,12 +31,15 @@ extension Slate {
         requestID = remote.requestId
         slateDescription = remote.description
 
+        var i = 1
         recommendations = NSOrderedSet(array: remote.recommendations.compactMap { remote in
             guard let remoteID = remote.id,
                   let recommendation = try? space.fetchRecommendation(byRemoteID: remoteID) ?? Recommendation(context: space.backgroundContext, remoteID: remoteID) else {
                 return nil
             }
             recommendation.update(from: remote, in: space)
+            recommendation.sortIndex = NSNumber(integerLiteral: i)
```

Comment:
Recommendations and slates are designed to be shown in a very specific order this lets us sort on that order in NSFetchedResultsController